### PR TITLE
Ct 148 puppet access login implementation

### PR DESCRIPTION
**Pull request by Dorin-Pleava**
WIP

### CODE REVIEW

File: app/puppet-access/login.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	if err != nil {
+		return err
+	}
+
+	if print {
+		fmt.Println(token)
+		return nil
+	}
+
+	return puppetAccess.saveToken(token)
```
Comment (Dorin-Pleava) [inline app/puppet-access/login.go:57]: Maybe there is a better solution than passing the print parameter to the Login method. Not sure if the printing should be handled by the SDK or the CLI

**Review by Dorin-Pleava — COMMENTED** [review]
Maybe there is a better solution than passing the print parameter to the Login method. Not sure if the printing should be handled by the SDK or the CLI https://github.com/puppetlabs/pe-sdk-go/pull/5/files#diff-457752cb2b162f0ebd928806aa29e8d1253dc16ee71fa1684b6789e680fa9286R45

File: app/puppet-access/login.go (hunk 1 above)
Comment (ciprianbadescu) [reply app/puppet-access/login.go:57]: Login could always return (string, error) and could be cli job to show it or not

Comment (ciprianbadescu) [reply app/puppet-access/login.go:57]: I fact I read documentation and we still need the flag, since we should not save it if `print` is true

Comment (gimmyxd) [reply app/puppet-access/login.go:57]: If we would handle this on CLI, can't we call the print/save action there?

Comment (Dorin-Pleava) [reply app/puppet-access/login.go]: I also think it's better that the output should not be handled by the SDK.
Login method will still receive the print parameter just so it does not save the token to disk.

Comment (ciprianbadescu) [reply app/puppet-access/login.go]: Yes, I think it is better that app/* is not directly calling token/*

**Comment by ciprianbadescu** [comment]
We could move api/puppet-access to app/puppet-access/api for better repo layout

**Review by ciprianbadescu — APPROVED** [review]

**Review by BobosilaVictor — APPROVED** [review]

**Pull request merged**